readthedocs/projects/base.py

```diff
@@ -0,0 +1,240 @@
+class BuildRequest(celery.worker.request.Request):
```

Comment:
It seems that I committed this file by mistake. It has the same content that `readthedocs/projects/tasks/base.py` --I should have failed with the file rename.